### ISSUE TITLE
Add support for a PDF export with speaker notes

### ISSUE THREAD

**Issue by hasalex** [opendevise/presentation-bespoke-starter#5]
I need a PDF export of my slides, in the same way than OOo Note's view. A4 pages,the first part of each page contains the slide itself and the second part contains the speaker notes.

I have a stylesheet that work with Chrome's "Export as PDF" feature, but I don't know how to switch to the stylesheet. Maybe a module, like the overview, that switches to a "slides with notes" view.

Any idea ?

**Comment by hasalex** [comment]
I switch to an other option, based on gulp. It's probably simpler, and it fits my personal need.

With the regular gulp commands, It uses the print.styl file, and when I add -n to the gulp command, it uses the print-notes.styl.

I've put the feature in my fork at https://github.com/hasalex/presentation-bespoke-starter/tree/export-pdf-notes. 

Tell me if you're interested with a PR.

**Comment by mraible** [comment]
@hasalex Good idea! I tried implementing this in my project and get the following error:

```
[14:12:11] Plumber found unhandled error:
 Error in plugin 'gulp-stylus'
Message:
    /Users/mraible/dev/presentations/microservices-for-the-masses/presentation/src/styles/print-notes.styl:4:41
   1| $real-margin-mm = 20mm
   2| 
   3| $ratio = 3.7795       // I don't know why this value, but it works
   4| $slide-height-mm = unit($slide-height / $ratio, 'mm')
----------------------------------------------^
   5| $slide-width-mm = unit($slide-width / $ratio, 'mm')
   6| 
   7| $zoom = unit( (210mm - 2 * $real-margin-mm) / $slide-width-mm, '')

cannot perform $slide-height / (3.7795)

Details:
    lineno: 4
    column: 41
    filename: /Users/mraible/dev/presentations/microservices-for-the-masses/presentation/src/styles/print-notes.styl
    stylusStack: 
```

I created a `variables.styl` and required it at the top of `main.styl`. Its contents are as follows:

```css
rem-calc(value)
  ((u = unit(value)) is "px") ? unit(round(value/16, 4), "rem") : value

$slide-width = 640px
$slide-height = 360px
```

**Comment by hasalex** [comment]
@mraible A small checklist:

- create variables.styl and print-notes.styl 
- add `@require './variables'` in main.styl
- remove `@require './print'` in main.styl
- add the dev dependency to gulp-concat in package.json
- add `concat = require('gulp-concat')` in gulpfile.js
- update the css task in gulpfile.js

With all that stuff, it seems to be working. Yet there's some layout to arrange...
I'll prepare a PR.

**Comment by hasalex** [comment]
@mraible BTW it should fix your problem with the regular export to PDF, as the gulp task add the print.styl or print-notes.styl file at the end of the stylesheet.

**Comment by mraible** [comment]
@hasalex That worked - thanks!

**Comment by mojavelinux** [comment]
This is looking really good. I'll do some review and see if we can get this integrated into this project.

Btw, if you use gulp-concat, don't forget to add it to package.json.

**Comment by hasalex** [comment]
Thanks for pointing me the package.json, @mojavelinux. I've updated the check list.
